### ISSUE TITLE
Error in EmailSender

### ISSUE THREAD

**Issue by at055612** [gchq/stroom#4175]
```
org.simplejavamail.internal.moduleloader.ModuleLoaderException: Batch module not found, make sure it is on the classpath (https://github.com/bbottema/simple-java-mail/tree/develop/modules/batch-module)

from stroom.analytics.impl.EmailSender.send(EmailSender.java:104)
```

**Comment by at055612** [comment]
Fixed in > 7.4-beta.1

**Comment by at055612** [comment]
Back ported to 7.3 fixed in >7.3-beta.14
